ScoutSuite/core/conditions.py

```diff
@@ -33,7 +30,7 @@ def pass_conditions(all_info, current_path, conditions, unknown_as_pass_conditio
         return True
     condition_operator = conditions.pop(0)
     for condition in conditions:
-        if condition[0] in condition_operators:
+        if condition[0] in ['and', 'or']:
```

Comment:
I'm curious, why this change?

Reply:
`condition_operators` was the only variable referenced in https://github.com/nccgroup/ScoutSuite/blob/master/ScoutSuite/core/__init__.py (the rest was dead code), and it was only referenced at that line in `conditions.py`, figured I'd remove the file as part of the clean up (less is more)

Reply:
Ok, nice! 💯